docs/user/examples/scale-to-zero-with-keda/assets/scaling-scenario.drawio

```diff
@@ -0,0 +1 @@
5d3WSvkDlbq/q5oSMxKBHA1xbRLETJEoq+m06M4SJv7gF4EVmK1zO6+LmPD8hicPXu/MkpLR+0dGGi9wbCRJm8gjZZQjXaAngoZleOqI+374bl3/EAuTV/L61P5B/ediwTZ/aE+JRSj2wgcQqXPioo7urPU48tJWyxxpnwKi4zlRTvDZyD5MNE5oX490MLMB+Iodlf9towswPvT4OGjPUMPakU5DMrygTAVd9/PO0VtamaLoP5orlU7VQP9Q71hdTrh3Xc3O45TJ6UvFnUb6XaoSyxS+x82LO3mTeB56SRSwhrmeJrPnzRUSHA9AKfTSlk3eJVLMODDv+11dly6wZhPt9yz5m2+THksNf10UCnULiJ/ansROfMuy1wkqCYu+kVRgSXLj5CeHAWNVhMaP/yLM6IwtrpJywqJ+sHvsY4zlVYpkUs+16yNHzDK2Bjbhv6BID/pf7drxLS/2IPCQIjrGdI2LG/lgZi38PMexM3ppp2OesPvuGzvn+c8EygAgJupsrCqfjKeq2OUTnxhLo8KeVAySGQu63mvCZozCA1ChOndUoHxOESPBqLBmzSJT9+3ZGkIxx0v2seoKhoU0o81ioZVip6aXZY60djsqC6s6c1wrz+RFYeJSjUqUdJQKSOuPOnOCWf6tLg8uTWtdkoLmdJ9BRBEdg5YkzbNshpPsQ6GYu/AREXGwVBEGqJKvC7gfJzhBFX2ZUMIAo1gC46C9AMEkrFxaDtehpkINWXi1vJLShShIPZMg75+MFAVp4qwwmfgKHVlCQSYv97QZiG+jpdUnacOzyggCj82etCCHHMJLUG8URLcwCPOwcRuWdpnRzk7WB0Nc8NzvP8ILRPquq0PNtD2BZeykPJCORWvQEeNhqiEmbwxVmp0Vr2poUpoGjV45eMPtSwtZh6vqV6W6Qxy9CAJZevPiCYj3ZLu9PTzHLQBo8hjawYC4tM5kmS3TONWpU+QLIx8YS37l0kJSZrYlxoEqZrLngMvn2UpS9w/tYaGxSnuSvHheOwf+MoLsGrSNTIvvwCPqyQNo/T0cZf0X7UxvQgVRnZwhv1U3KanVTk7sb/jT05UIuG7Z+/mtJ+SWFMBq/spAJFoM6hX2u27knFieS+8cCa1XiD9vT1hr8lw1amfOJb5jzWBYalrtUp/g1XDTPq7S07bt8zOpv+xZgOp0hMUknT1wWwtOHZWzD+bu7BBBvICNrTJhyftgasnLpz3+pFioczQddFKvOaReT5o+7xEcW5fwa7+U66VM4Sh7RxiAlJmtkNeOsdCNxPWVadbSANXMmyCvhJXU6DX931mWQpjDOgJ5CQfa3ko7vmVkeFyybgW3U15fJGXZBFrBsF3b9Q7/sxveca4B4JyxxNTu174xDe2NT4O08bAg2ZpWLYdS/2E0ewTrRv9hnd2P2xjzDI4EdBA/EnhTAfi4r+qJwgD0qzaIXrmSxX2gvh8FsbKWC8l8vyu32bXZM1QJxIblJt6ZM/LFze+51uKsjxMB3g3tgQXizPjuChyTmw1ch85/IKZjEYBS+lkFeT+BiQII6FRe9UTALtrOYwhdg+62XhfSNbWrz0HnM3pWkVZsj/+LBnjClK4MpFtruJXtWc8SGwZF96367U+RCqwyN06zOn2GFd//OTLvGhsVwueYLJpWWatYgZIdg6SByjleIqUW3Jenr35xic9tGHI+HZMizblwV4pQcHBpooCj3/rPsfkxpL7EVOVOTqmRSCn0DIB3i9lq0+cux7ILb6n24AqdgcYU27lPxAgnOQPSsM9O961hFrb/79BfRTWIWnWlUC65GfUZKtpztbmEuEHJOj83UwsNBYFQ+08htq8/TdpCyqO70cV4porcrZSVONptFcgmxIA7OAU6b5wW9ZhUJ7pKssfcQT6nfjx1xcbKxzcEav72Yfsryrb5UVLMjg2A9XrC8Ctbn8R1JqpkPkjgid6nEyO7rBUQmV0Gg8o8SNKwPYT7FpHBK8D9PzjAraCXJ5YMPKzdMktfubP6oG+GFccBqeDGcoAGconwdemllqV+KWAKsJpnhgvb7Np9YeidgwPXRKe4c5zlrbn99eLKnV3B4w8DylDwllC9HiOOUjcttCm+/ikH/klqe+h5ls+lVIKQdeNE9HqGOdF/FEUdg0FTmYeNqOo5j4Eamkcazw2RGJ6MnQ32s1yoVcLsZcUUNeLhoKytg3H1p2IhVKj4tkjTMJyGCTWLjqM3jkf0Cxi8jxmvNrdM+GwKsS7/UkeXS+mJAtFmAWxA8SyjBdOmxrepmVjM0L4dc6cQ0VtkPs7BNSbX8afGrxCP5sLvNJLIfA2zxuQoKPVyQhYNlGdSmkBfQ8X9L2jWFuB88xd63xnB/kwxT9z7ukezFCBIv+H0sKS4NBhf4UkBaB8/gGYV0j4o0OuU0mNrULXRFEbdWRJYwHijscxK0AzJ+6XO8n533sz42Hh6eIOIee0O2A+tjeW4dNxZpRAsR5A9zcsaWIAeK+qvMKLJ7FPw6jLHEOp9OfkZethiBry/ewuVE77ESfB/dBy/3cco6zqvNr0omklgL/HIJZRH33ftOvvsWy8xlS+zbddO6tPs1omLJf4fcI68kKFwd9sVsjTt5jckjs3993knD2JxTkdhajmCJDH+GDdr0kyMMX0Bb7VZqBnmVKfKwOclwtPYOhlMVSM+df/Yvqs1KV7JrAlZLO7aD51hmvm+mlgR5t/u3DR/F2fsmXW9tP6l3HGULvlHo73MhDcSCvN+vTL3JCnTJWBR/mLeTICgeii+OlhhnGJUHl/P9zJm64V1yf70omBHdyvvzyP8+K2xXdLULAykqhkuwpvjm3+b367jdAyfWgfgCIFnA7o47ubd16+WNzDvUUskYFJM4xg+o68kmN81G5MdTi/9ILqi6AS78+hf41ODkDPj7f3JuiACX9v7rgS+U+jfk/z47hBD/9p/Hnf/liBz2P7gvEvj3CVzI/l9HGcHdL8ZUgHOjwr8D</diagram></mxfile>
```

Comment:
this doesn't match our diagram guidelines :/

Reply:
I'll work on it.